static/admin-widget-style.css

```diff
@@ -23,7 +23,7 @@
 }
 
 .button-style {
```

Comment:
Discovered that this CSS rule has border color defined in both `border` and `border-color` properties, but only latter is applied. Please correct this also.